Formula/mingw-w64.rb

```diff
@@ -0,0 +1,177 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows with GCC."
+  homepage "https://mingw-w64.org/"
+  # https://sourceforge.net/p/mingw-w64/wiki2/Cross%20Win32%20and%20Win64%20compiler/
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v5.0.1.tar.bz2"
+  sha256 "9bb5cd7df78817377841a63555e73596dc0af4acbb71b09bd48de7cf24aeadd2"
+
+  option "with-multilib", "Compile x86_64 compiler without multilib support"
```

Comment:
I've been switching it back and forth since it currently still doesn't work. I'm going to comment the option out right now but leave all of the messages in the formula since I think someone will eventually get it working